src/actions/mod.rs

```diff
@@ -175,10 +176,12 @@ impl ActionHandler {
 
                         debug!("reload analysis: {:?}", project_path_clone);
                         let cwd = ::std::env::current_dir().unwrap();
-                        if let Some(new_analysis) = new_analysis {
-                            analysis.reload_from_analysis(new_analysis, &project_path_clone, &cwd, false).unwrap();
-                        } else {
+                        if new_analysis.is_empty() {
```

Comment:
It is rare for there to be no save-analysis info, even if we fail to compile, so I expect this will not be a problem. In any case, it should be resolved once we build successfully, so it should only be temporary.